copyright/copyright.js

```diff
@@ -0,0 +1,109 @@
+/* * *  *  * *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  * */
+/* Copyright (c) 2017 Mobify Research & Development Inc. All rights reserved. */
+/* * *  *  * *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  * */
+
+const glob = require('glob')
+const fs = require('fs')
+const path = require('path')
+
+const red = '\x1b[31m'
+const green = '\x1b[32m'
+const yellow = '\x1b[33m'
+const magenta = '\x1b[35m'
+const cyan = '\x1b[36m'
+const blackBG = '\x1b[40m'
+const defaultBG = '\x1b[49m'
+const defaultFG = '\x1b[39m'
+
+const args = process.argv.filter((arg) => {
+    return !/node|copyright/.test(arg)
+})
+
+const copyright = {
+    lintMode: false,
+    langs: {},
+    run() {
+        this.buildSupportedExtensions()
+        const processedGlobs = args.map((folder) => {
+            return new Promise((resolve) => {
+                glob(`${folder}`, (err, file) => {
+                    resolve(file[0])
```

Comment:
Each matched `file` that's inside `glob()` callback is an Array of length 1
![screen shot 2017-05-04 at 10 51 27 am](https://cloud.githubusercontent.com/assets/16109837/25717597/cceaab8a-30b7-11e7-8d81-e303be2620ed.png)

From `node-glob`:

```javascript
var glob = require("glob")

// options is optional
glob("**/*.js", options, function (er, files) {
  // files is an array of filenames.
  // If the `nonull` option is set, and nothing
  // was found, then files is ["**/*.js"]
  // er is an error object or null.
})
```

So it seems like this should be an array of all of the matches - but instead the functionality is that the callback is fired 1 time for each matched file - and `file` is an array of length 1 🤔